api/v1/storagecluster_types.go

```diff
@@ -216,6 +216,12 @@ type ManageCephCluster struct {
 
 	// Whether to allow updating the device class after the OSD is initially provisioned
 	AllowDeviceClassUpdate bool `json:"allowDeviceClassUpdate,omitempty"`
+
+	// monTimeout specifies the timeout for Ceph monitor failover, mapping to
+	// cephCluster.spec.healthCheck.daemonHealth.mon.timeout in Rook Ceph.
+	// The default is 10 minutes. This field allows customization or disabling
+	// failover by setting the value to 0.
+	MonTimeout string `json:"monTimeout,omitempty"`
```

Comment:
From the discussion in https://issues.redhat.com/browse/DFBUGS-565 I thought we agreed to go with the full spec instead of defining a new value, something like this:
```suggestion
	HealthCheck rookCephv1 CephClusterHealthCheckSpec `json:"monTimeout,omitempty"`
```

Reply:
done